test/pkg/environment/azure/environment.go

```diff
@@ -40,16 +43,55 @@ const (
 
 type Environment struct {
 	*common.Environment
-	Region string
+	AzureClients
+	Vars
```

Comment:
I don't love having multiple unnamed fields in a struct. Prefer keeping it to 1 max, but maybe that's just personal preference.

Reply:
Removed them all and flattened the data structure.